qml/pages/BurdenModulePage.qml

```diff
@@ -150,16 +150,39 @@ BaseTabPage {
                             top: burdenIntrospection.ComponentInfo[parNominalRange.controlPropertyName].Validation.Data[1];
                             decimals: FT.ceilLog10Of1DividedByX(burdenIntrospection.ComponentInfo[parNominalRange.controlPropertyName].Validation.Data[2]);
                         }
+
+
+                        // The current Burden does not need a Rangefactor
+                        // Therefore it is always 1 and the box is invisible
                         ZVisualComboBox {
                             anchors.left: parent.right
                             anchors.leftMargin: 8
                             anchors.top: parent.top
                             anchors.bottom: parent.bottom
                             width: page.width*0.09;
                             contentRowHeight: height*1.2
+                            visible: {
+                                if(page.isVoltagePage){
+                                    return true;
+                                }else{
+                                    return false
+                                }
+                            }
 
```

Comment:
Nitpicker's alert: Why not just
visible: page.isVoltagePage

Reply:
Good question no answer